src/SamplesApp/SamplesApp.UITests/TestFramework/IsExternalInit.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace System.Runtime.CompilerServices;
 
-#if !NET5_0 && !NET6_0
+#if !NET5_0 && !NET6_0_OR_GREATER
```

Comment:
```suggestion
#if !NET5_0_OR_GREATER
```

Reply:
`NET5_0_OR_GREATER` doesn't exist ^^

Reply:
@dr1rrb I thought it should exist 👀 

At least it does in another project, but what would make it available in one but not the other?

https://github.com/unoplatform/uno/blob/896018279fd913fc88ae1e5eb10cb3e1b6f7f1c3/src/SamplesApp/SamplesApp.UnitTests.Shared/Controls/UnitTest/UnitTestsControl.cs#L89

Reply:
Well ... I only trusted what said @jeromelaban 🤪